test/bot.test.ts

```diff
@@ -0,0 +1,373 @@
+      }),
+    });
+    // Load our app into probot
+    probot.load(myProbotApp);
+  });
+
+  describe("Issue Comment", async function() {
+    it('should create a comment on the issue if repo owner tagged gitpoap-bot and contributors on an issue comment', async (done) => {
+      const githubAPIMock = nock('https://api.github.com')
+        // Test that we correctly return a test token
+        .post('/app/installations/29153052/access_tokens')
+        .reply(200, {
+          token: 'test',
+          permissions: {
+            issues: 'write',
+          },
+        })
+  
+        // get github login ids
+        .get('/users/test1')
+        .reply(200, {
+          "id": 1,
+        })
+        .get('/users/test2')
+        .reply(200, {
+          "id": 2,
+        })
+        .get('/users/test3')
+        .reply(200, {
+          "id": 3,
+        })
+  
+        // Test that a comment is posted
+        .post('/repos/gitpoap/gitpoap-bot-test-repo/issues/25/comments', (body: any) => {
+          done(expect(body).toMatchObject(issueCreatedBody));
+          return true;
+        })
+        .reply(200);
+  
+      // Test response from gitpoap api
+      const gitpoapAPIMock = nock(`${process.env.API_URL}`)
+        .post(`/claims/gitpoap-bot/create`)
+        .reply(200, {
+          newClaims
+        });
+  
+      // Receive a webhook event
+      await probot.receive({ name: 'issue_comment', payload: issueCommentPayload });
+  
+      expect(githubAPIMock.activeMocks()).toStrictEqual([]);
+      expect(gitpoapAPIMock.activeMocks()).toStrictEqual([]);
+    });
+  
+    it('should create a comment on the PR if repo owner tagged gitpoap-bot and contributors on an PR comment', async (done) => {
+      const githubAPIMock = nock('https://api.github.com')
+        // Test that we correctly return a test token
+        .post('/app/installations/29153052/access_tokens')
+        .reply(200, {
+          token: 'test',
+          permissions: {
+            issues: 'write',
+          },
+        })
+  
+        // get github login ids
+        .get('/users/test1')
+        .reply(200, {
+          "id": 1,
+        })
+        .get('/users/test2')
+        .reply(200, {
+          "id": 2,
+        })
+        .get('/users/test3')
+        .reply(200, {
+          "id": 3,
+        })
+  
+        // Test that a comment is posted
+        .post('/repos/gitpoap/gitpoap-bot-test-repo/issues/25/comments', (body: any) => {
+          done(expect(body).toMatchObject(issueCreatedBody));
+          return true;
+        })
+        .reply(200);
+  
+      // Test response from gitpoap api
+      const gitpoapAPIMock = nock(`${process.env.API_URL}`)
+        .post(`/claims/gitpoap-bot/create`)
+        .reply(200, {
+          newClaims
+        });
+  
+      // Receive a webhook event
+      await probot.receive({ name: 'issue_comment', payload: prCommentPayload });
+  
+      expect(githubAPIMock.activeMocks()).toStrictEqual([]);
```

Comment:
is it possible to use `https://github.com/nock/nock#isdone` here instead of checking that `activeMocks` is empty? 

i understand the logic, but might be clearer to say "X was done" instead of "There is Y left to do and Y is empty". thoughts?